### DISCUSSION TITLE
Nginx Proxy Manager - v2.11.1 (latest) buggy from source

### DISCUSSION THREAD

**Discussion by jngutierrez** [tteck/Proxmox#2432] [General]
NPM v2.11.1 has an offending bug related to custom locations that render host proxies unusable. It might be a good idea to modify the script to install the latest stable version (v2.10.4) until the issue is resolved. Refer to:

[NPM Issue 3474](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3474
)
[NPM Issue 3484](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3484)
[NPM Issue 3508](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3508)

I tried to modify the helper script to install a custom version but still the script installs the latest.

**Comment by dazeb** [comment]
I came here to say the same, i installed it in docker but its just not the same for me. 

Hope this can be fixed!

**Reply by tteck** [reply]
Added option to install v2.10.4 https://github.com/tteck/Proxmox/commit/bfa4cd815e6af95f1a81421ef1be0eab5b58dccb

**Reply by jngutierrez** [reply]
I confirm this works flawlessly. **Thank you very much.**

**Comment by dilighiani** [comment]
Hi @tteck ,
I want to thank you for your awesome tools! I just discovered yesterday.

Today I wanted to install NginxPM and I receive below error ONLY if I choose Custom installation, but if I choose Standard installation wizard (non-custom), the setup finishes successfully.

|Setting up python3-certbot-dns-cloudflare (2.0.0-1) ...
-Processing triggers for man-db (2.11.2-1) ...
|Processing triggers for libc-bin (2.37-0ubuntu2.2) ...
|error: externally-managed-environment
× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
..
..
[ERROR] in line 42: exit code 0: while executing command $STD pip3 install certbot_dns_porkbun

**Reply by tteck** [reply]
Ubuntu?

**Reply by tteck** [reply]
> - If the default Linux distribution is not adhered to, script support will be discontinued.

**Reply by dilighiani** [reply]
Yes, I tried initially with Ubuntu as it is my preferred choice and as long as I had that option.
I can confirm that I tried again now choosing Debian and the script finished OK and everything's fine. 👍
Thank you for your very quick response!
